src/coreclr/src/pal/tests/palsuite/README.txt

```diff
@@ -44,17 +44,6 @@ The results are logged to %ROTOR_DIR%\tests\pvtResults.log.
 
 3. ENVIRONMENT VARIABLES AND AUTOMATED TESTING SPECIFICS
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    
-3.1 PAL_DISABLE_MESSAGEBOX
-
-For automated testing on the WIN32 PAL, the following environment
-variable is useful to prevent pop-up message boxes from interupting the
-execution of the MessageBoxW test cases:
-
-set PAL_DISABLE_MESSAGEBOX=1  
-
-
-3.2 Other Notes
 
```

Comment:
Why is this removed?

Reply:
It was to address this comment: https://github.com/dotnet/runtime/pull/31659#discussion_r374620078